packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,63 +1,143 @@
 # @fluentui/react-infobutton Spec
 
+Convergence epic issue: [#25062](https://github.com/microsoft/fluentui/issues/25062)
+
 ## Background
 
-_Description and use cases of this component_
+An InfoButton provides a way for users to get more information about a particular UI element. It is a button that displays an icon and when clicked a Popover displays the additional information.
+
+Because the Popover isn't always visible, it should not contain information that people must know in order to complete the field.
+
+### Anatomy
+
+![Anatomy](./etc/images/anatomy.png)
```

Comment:
This image shows the popover in the "above-start" position. Will `InfoButton` support positioning of the popover?

Reply:
Yes, that will be done through the root as it takes the popover props as well. The updated types now have the Popover props.